### PR TITLE
Implement Upvote/Downvote Functionality in FE

### LINKED ISSUE #36: Add up and downvote buttons to UI and connect to graphQL mutation
- [ ] needs access to method_results ID
- [ ] store in redux store
- [ ] up vote button 
- [ ] down vote button
- [ ] add mutation query (method_result_id, +1/-1)
   - argument needs to include id and if it's an up or downvote

### PR DESCRIPTION
**Pull request by DanielEFrampton**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project
- [x] Added to Readme (nothing changed)
​
### Issues Resolved
Resolves #36 
​
### Problem Addressed
No user-facing ability to interact with upvote/downvote functionality implemented in BE.
​
### Solution Implemented
- Changed ResultsContainer into a class component
- Wrote function to interact with createVote mutation when clicking upvote/downvote buttons
​
### Screenshots (optional)
​

### Other Notes

### CODE REVIEW

File: src/components/ResultCard/ResultCard.jsx [hunk 1]
```diff
@@ -28,11 +28,13 @@ export const ResultCard = ({rating, method}) => {
...
       <section className='user-vote-section'>
-        <button className='vote-btn'>
+        <button className='vote-btn' onClick={(event) =>
+        handleVote(resultId, event, 'down')}>
           This wasn't helpful
           <span className='down-vote'></span>
         </button>
-        <button className='vote-btn'>
+        <button className='vote-btn' onClick={(event) =>
+        handleVote(resultId, event, 'up')}>
```
Comment (mintona) [inline src/components/ResultCard/ResultCard.jsx:37]: Yes! Voting buttons. Nice.

File: src/containers/ResultsContainer.jsx [hunk 2]
```diff
@@ -1,46 +1,71 @@
...
-const ResultsContainer = (props) => {
-  let results = props.results.map(result => {
-    return result.translations.map(res => <ResultCard rating={res.weightedRelevancyRating} method={res.method}/>)
-  })
+class ResultsContainer extends Component {
+  constructor(props) {
+    super();
+  }
+
+  handleVote = async (resultId, event, type) => {
```
Comment (VeeAndrade) [inline src/containers/ResultsContainer.jsx:17]: Awesome job creating this function!

File: src/queries/index.js [hunk 3]
```diff
@@ -34,3 +35,21 @@ export const GET_TRANSLATION =
...
+       id
+       name
+       description
+       syntax
+       snippet
+       docsUrl
+     }
+   }
+ }
+`
```
Comment (mintona) [inline src/queries/index.js:55]: This is great.

File: src/reducers/results.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 export const results = (state = [], action) => {
   switch(action.type) {
     case 'SET_RESULTS':
-      return [action.results]
+      return action.results
```
Comment (mintona) [inline src/reducers/results.js:4]: Great sleuthing.

**Review by mintona — APPROVED** [review]
Really nice work on this everyone! Very excited we got this in time.

**Review by VeeAndrade — APPROVED** [review]
Glad we got our stretch goal done!

**Pull request merged**